lib/kokusei_chousa.py

```diff
@@ -0,0 +1,21 @@
+"""
+国勢調査です
+鯖民の通常ユーザー/BOTの数を統計します
+例によって関数です
+"""
+
+import discord
+
+NUMBER_TEXT = "***†只今の限界開発鯖の人口†***\n```\n・通常ユーザー: {} 人\n・Botユーザー: {} 人\n・Bot率: {}%```"
+
+
+async def number(channel: discord.TextChannel):
+    bot_number = 0
+    user_number = 0
+    for member in channel.guild.members:
+        if member.bot:
+            bot_number += 1
+        else:
+            user_number += 1
+    bot_ratio = float(bot_number) / (float(bot_number) + float(user_number))
```

Comment:
```suggestion
    bot_ratio = bot_number) / (bot_number + user_number)
```
floatいらないはずだで
テストしてないので動かなかったらすまんこ

Reply:
できますねぇ！